pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -382,6 +383,26 @@ func (dswp *desiredStateOfWorldPopulator) createVolumeSpec(
 			pvcSource.ClaimName,
 			pvcUID)
 
+		// Error if a container doesn't have volumeDevices but the volumeMode of PVC is Block
+		if len(devicesMap) == 0 && volumeMode == v1.PersistentVolumeBlock {
```

Comment:
Can this check be merged with the 2nd check?  You just need to check if there's an entry in the map && mode is block

Reply:
will do.